docs/compilers/CSharp/Compiler Breaking Changes - post VS2017.md

```diff
@@ -119,3 +119,8 @@ Example:
    }
    	```  
 
+- https://github.com/dotnet/roslyn/issues/25450 In C# 7.3 we add new restrictions on the use of the `default` expression in preparation for the planned addition of further pattern-matching features in C# 8.0:
```

Comment:
Was the change made in 15.7? I think it would be good to specify that version.

Reply:
OK

Reply:
just a thought: I think saying "In C# 7.3 we add new restrictions" is a little misleading because it makes it sound like this change is only going to affect you if you're using language version 7.3 or newer, but that's not the case as far as I know. It affects 7.1 as well, doesn't it?